staging/src/k8s.io/kube-aggregator/pkg/controllers/status/available_controller.go

```diff
@@ -454,6 +456,12 @@ func (c *AvailableConditionController) deleteAPIService(obj interface{}) {
 
 // there aren't very many apiservices, just check them all.
 func (c *AvailableConditionController) getAPIServicesFor(obj runtime.Object) []*apiregistration.APIService {
+	var svc *v1.Service
+	if svc, ok := obj.(*v1.Service); ok {
```

Comment:
this doesn't handle endpoint objects, which are far more frequently updated

Reply:
Looking at ./staging/src/k8s.io/api/core/v1/types.go , there are several types related to endpoint.

Endpoints
EndpointSubset
EndpointAddress
EndpointsList

I assume you meant v1.Endpoints